workbench-core/user-management/src/plugins/cognitoUserManagementPlugin.ts

```diff
@@ -28,6 +29,7 @@ export class CognitoUserManagementPlugin implements UserManagementPlugin {
   private _userPoolId: string;
   private _aws: AwsService;
   private _tempRoleAccessSettings?: { ddbService: DynamoDBService; ttl?: number };
+  public readonly userRoleLimit: number = 25;
```

Comment:
Reason for this number?

Reply:
Please refer to the ticket `issues/P88136850`: we are enforcing only 25 groups at this point